analysis/topEFT/datacard_maker.py

```diff
@@ -194,7 +194,9 @@ def export1d(h, name, cat, fout):
                 for syst,histo in h.items():
                     if syst == 'nominal':
                         fout[name+cat] = hist.export1d(histo)
-                    elif self.do_nuisance and name not in self.syst_special and 'flips' not in name:
+                    elif self.do_nuisance and name not in self.syst_special:
+                        if 'fakes' in name and 'FF' not in syst: continue # Only processes fake factor systs for flips
```

Comment:
This is a very minor question, but just to avoid confusion, should the comment say "fakes" here not "flips"?

Reply:
Right, I'll fix it now.

Reply:
Thanks